.idems_app/deployments/plh/default.workflows.ts

```diff
@@ -0,0 +1,68 @@
+import { IDeploymentWorkflows } from "data-models/workflow.model";
+
+/*************************************************************************************
+ * Workflows
+ ************************************************************************************/
```

Comment:
This is an eample of the current under-construction workflows. The `sync` workflow shows examples of both recursively calling additional workflows, as well as standalone tasks.